decidim/decidim
# Enable viewing proposals in grid mode

**Closing this PR because it was based on the wrong branch. A new PR will be created from the correct base branch.**


#### :tophat: What? Why?

This pull request implements a grid view mode for proposal viewing. The new mode enhances visual appeal and engagement by displaying proposals as cards with accompanying images. This improvement addresses the issue of the current proposal viewing mode lacking visual appeal and engagement.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12797 

#### Testing

- Access the user interface and navigate to the proposal view.
- Verify there is an option to toggle between list and grid views.
- Switch between list view and grid mode, ensuring proposals are displayed as cards with images in grid mode.
- Confirm each card shows relevant information along with images, and that a placeholder image appears when no image is attached.
- Test the grid mode on various devices and screen sizes to ensure responsiveness and maintain visual appeal.
- As an admin, enable the "Allow Attachments" setting and verify the display text: "By enabling this option, proposals will default to grid mode, and the first image will appear on the card."

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
